homeassistant/components/netatmo/sensor.py

```diff
@@ -50,7 +54,7 @@
         DEVICE_CLASS_TEMPERATURE,
     ],
     "co2": ["CO2", CONCENTRATION_PARTS_PER_MILLION, "mdi:molecule-co2", None],
-    "pressure": ["Pressure", "mbar", "mdi:gauge", None],
+    "pressure": ["Pressure", PRESSURE_MBAR, "mdi:gauge", DEVICE_CLASS_PRESSURE],
```

Comment:
We don't set a custom icon if we set a device class.

Reply:
Oh, fair point. I'll fix that.

Reply:
@MartinHjelmare Pushed to #38859